### DISCUSSION TITLE
Welcome to hashify Discussions!

### DISCUSSION THREAD

**Discussion by dwin** [dwin/hashify#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tjanyl** [comment]
Hi @dwin  Been enjoying the Hashify API a lot and also doing some tests with it on Microsoft Power Automate. Thanks for the great work!

I had a question regarding chunking. Since Power Automate is not able to send large files, it needs to do so by chunks. Does the Hashify API support chunking? Or is there another way to send large files via API?

Thank u so much in advance! Your work is much appreciated.
